File: problema01/sol_java/problema01/src/problema01/Problema01.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package problema01;

import java.util.Locale;
import java.util.Scanner;

/**
 *
 * @author USUARIO
 */
public class Problema01 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here

        Scanner entrada = new Scanner(System.in);
        entrada.useLocale(Locale.US);

        int A;
        int B;
        int C;
        String a = "Triángulo Rectángulo";
        String b = "Triángulo Obtusángulo";
        String c = "Triángulo Acutángulo";
        String triangulo;

        System.out.println("Ingresar el valor del primer Ángulo");
        A = entrada.nextInt();

        System.out.println("Ingresar el valor del segundo Ángulo");
        B = entrada.nextInt();

        System.out.println("Ingresar el valor del tercer Ángulo");
        C = entrada.nextInt();

        if ((A == 90) || (B == 90) || (C == 90)) {
            triangulo = a;
        } else {
            if (((A > 90) && (A < 180)) || ((B > 90) && (B < 180))
                    || ((C > 90) && (C < 180))) {
                triangulo = b;
            } else {
                triangulo = c;
            }
        }
        System.out.printf("Los datos ingresados pertenecen a un:\n%s\n"
                , triangulo);
    }

}
        

File: problema01/sol_python/problema01.py
a = "Triángulo Rectángulo"
b = "Triángulo Obtusángulo"
c = "Triángulo Acutángulo"
A = int(input ("Ingresar el valor del ángulo A: "))
B = int(input("Ingresar el valor del ángulo B: "))
C = int(input("Ingresar el valor del ángulo C: "))
if (A == 90) or (B == 90) or (C == 90):
            triangulo = a
else:
    if (A > 90) and (A < 180) or (B > 90) and (B < 180) or (C > 90) and (C < 180):
             triangulo = b
    else:
        triangulo = c
print("Los datos ingresados pertenecen a un:", triangulo)